app/controllers/Application.java

```diff
@@ -2046,6 +2050,26 @@ public static Result updateTuneinDetails() {
     return ok(new Gson().toJson(parent));
   }
 
+  public static Result recordExceptionfingerprintingFeedback() {
+    JsonNode requestBodyRoot = request().body().asJson();
+    String flowExecUrl = requestBodyRoot.path(FLOW_EXEC_URL).asText();
+    boolean isHelpful = requestBodyRoot.path(IS_HELPFUL).asBoolean();
+    logger.info("Updating the Helpful property for ExceptionFingerprinting result " + flowExecUrl);
+    if (Strings.isNullOrEmpty(flowExecUrl)) {
+      return badRequest("Flow Execution Url is not provided");
+    }
+    List<JobsExceptionFingerPrinting> exceptionFingerPrintingResult = JobsExceptionFingerPrinting.find.select("*")
+        .where()
+        .eq(JobsExceptionFingerPrinting.TABLE.FLOW_EXEC_URL, flowExecUrl)
```

Comment:
I think with current logic user will not be able to provide the feedback for individual jobs ( it may happen the EF is good for one job and bad for another ) , If this assumption is correct , can u please write the todo comment in the code about your assumption and we can create jira for it be done as next PR

Reply:
Add ToDo and create JIRA.